ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,28 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  name: '4.0'
+  namespace: ocp
+base_images:
+  base:
+    name: '4.0'
+    namespace: ocp
+    tag: base
+canonical_go_repository: github.com/operator-framework/community-operators
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+build_root:
+  project_image:
+    dockerfile_path: Dockerfile
+    context_dir: ci.Dockerfile
+tests:
+- as: scorecard
+  commands: eval $(scripts/ci/operator-env) && scripts/ci/test-pr openshift
```

Comment:
Is there a pr for this yet in the community-operators repo? If this gets merged it prevents us from merging there if this isn't passing.

Reply:
https://github.com/operator-framework/community-operators/pull/71